agimus_controller_ros/agimus_controller_ros/agimus_controller.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+import rclpy
+from rclpy.node import Node
+from rclpy.time import Time
+from rclpy.qos import qos_profile_system_default
+from rclpy.qos_overriding_options import QoSOverridingOptions
+import numpy as np
+from copy import deepcopy
+import time
+from threading import Lock
+from std_msgs.msg import Duration, Header
+from linear_feedback_controller_msgs.msg import Control, Sensor
+import atexit
+
+from agimus_controller_ros.ros_np_multiarray import to_multiarray_f64
+from agimus_controller.trajectory_buffer import TrajectoryBuffer
+from agimus_controller.robot_model.panda_model import get_task_models
+from agimus_controller.utils.pin_utils import get_ee_pose_from_configuration
+from agimus_controller.mpc import MPC
+from agimus_controller.ocps.ocp_croco_hpp import OCPCrocoHPP
+from agimus_controller_ros.sim_utils import convert_float_to_ros_duration_msg
+
+
+class AgimusController(Node):
+    def __init__(self, node_name: str = "agimus_controller_node") -> None:
+        super().__init__(node_name)
+        self.traj_buffer = TrajectoryBuffer()
+        self.rmodel, self.cmodel, self.vmodel = get_task_models(
```

Comment:
model will be coming from outside, maybe we need to sync with Vamsi on how to access it